Sources/Common/SecurityOrigin.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  URLExtension.swift
```

Comment:
Please replace this with SecurityOrigin.swift, otherwise SwiftLint will complain once we enable it on all files.